src/include/processor/operator/persistent/parquet_column_writer.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include "processor/operator/persistent/csv_parquet_writer.h"
+#include <parquet/api/writer.h>
+
+namespace kuzu {
+namespace processor {
+
+// Kuzu column represents a single column in Kuzu, which may contains multiple Parquet columns
+// writeColumn will be called for each Kuzu column.
+class ParquetColumnWriter {
+public:
+    ParquetColumnWriter(int totalColumns, std::vector<int> maxDefinitionLevels)
+        : maxDefinitionLevels(maxDefinitionLevels), totalColumns(totalColumns){};
+
+    void writeColumn(
+        int kuzuColumn, common::ValueVector* vector, parquet::RowGroupWriter* rowWriter);
```

Comment:
I think we can just simplify `kuzuColumn` as `columns`. Ditto on the above class comment. 
```
A single column in Kuzu may contain multiple Parquet columns, for example, ...
```

Also, `rowWriter` to `rowGroupWriter`.